app/views/application/index.html.erb

```diff
@@ -1,309 +1,69 @@
-<div class="container-fluid">
-  <div class="row">
-    <div class="col-md-12 navigation">
-      <div id="logo">
-        <center>
-          <h2 style="padding-top:5px;">
-            <%= image_tag "sfbbo-logo.jpg" %>
-            <%= link_to 'BIRDING THE BAY TRAIL', root_path, {:style=>'color:#000000; text-decoration:none;', } %>
-          </h2>
-        </center>
-      </div>
-    </div>
-  </div>
-
-  <div class="row">
-    <div class="col-md-2">
-      <div class="row">
-        <h5>Nearby Locations</h5>
-      </div>
-      <div class="row">
-        <%= link_to 'Show All Locations', locations_path%>
-      </div>
-      <div class="row">
-        <div id="nearby-locations-sidenav" class="sidenav">
-          <% @locations_full.each do |location| %>
-            <div id="location-card" class="card">
-              <div class="card-body">
-                <h5> <%=link_to location.name, location.websiteURL %> </h5>
-                <p class="card-text" style="font-size: 12px;"><h5><%= number_with_precision(hav_distance([location.latitude.to_f, location.longitude.to_f], @current_location), precision: 2) %> mi</h5></p>
+<main>
+  <%= javascript_include_tag 'index' %>
+  <%= render "header" %>
+
+    <div class="row">
+      <div class="col-md-2">
+        <div class="row">
+          <h5>Nearby Locations</h5>
+        </div>
+        <div class="row">
+          <%= link_to 'Show All Locations', locations_path%>
+        </div>
+        <div class="row">
+          <div id="nearby-locations-sidenav" class="sidenav">
+            <% @locations_full.each do |location| %>
+              <div id="location-card" class="card">
```

Comment:
It looks like you have a DIV with an ID inside a .each- wouldnt that create multiple elements with same ID? An ID should be unique to a page.